docs/multisigs/emergency-brakes.md

```diff
@@ -364,7 +364,7 @@ L2ERC20TokenBridge: Mode:[`0xb8161F28a5a38cE58f155D9A96bDAc0104985FAc`](https://
 
 **Contracts and Roles:**
 
-L2ERC20TokenBridge: Zircuit:[`0x9Bff79BF7226cB5C16d0Cca9c1dc60450feE560d`](https://safe.zircuit.com/settings/setup?safe=zircuit-mainnet:0x9Bff79BF7226cB5C16d0Cca9c1dc60450feE560d)
+L2ERC20TokenBridge: Zircuit:[`0x7721F53d153Ae3CF937605fF1Bbb7D51B14E7902`](https://explorer.testnet.zircuit.com/address/0x7721F53d153Ae3CF937605fF1Bbb7D51B14E7902)
```

Comment:
@TheDZhon this seems to be for the Testnet?

Reply:
yeah, good catch! :+1: fixed